client/src/config/app/locale/en/en.json

```diff
@@ -163,7 +165,52 @@
       "visit_page": "Visit page",
       "view_interactive_resource": "$t(actions.view) Interactive Resource",
       "see_all_issues": "$t(actions.see_all) $t(glossary.issue_other)",
-      "see_all_volumes": "$t(actions.see_all) $t(glossary.volume_other)"
+      "see_all_volumes": "$t(actions.see_all) $t(glossary.volume_other)",
+      "view_interactive_resource": "$t(actions.view) Interactive Resource"
+    },
+    "search": {},
+    "glossary": {
+      "project_one": "project",
+      "project_other": "projects",
+      "project_collection_one": "project collection",
+      "project_collection_other": "project collections",
+      "text_one": "text",
+      "text_other": "texts",
+      "text_section_one": "text section",
+      "text_section_other": "text sections",
+      "journal_one": "journal",
+      "journal_other": "journals",
+      "volume_one": "volume",
+      "volume_other": "volumes",
+      "issue_one": "issue",
+      "issue_other": "issues",
+      "article_one": "article",
+      "article_other": "articles",
+      "resource_one": "resource",
+      "resource_other": "resources",
+      "resource_collection_one": "resource collection",
+      "resource_collection_other": "resource collections",
+      "reading_group_one": "reading group",
+      "reading_group_other": "reading groups",
+      "annotation_one": "annotation",
+      "annotations_other": "annotations",
+      "collection_one": "collection",
+      "collection_other": "collections",
+      "contributor_one": "contributor",
+      "contributor_other": "contributors",
+      "item_one": "item",
+      "item_other": "items",
+      "progress_bar": "progress bar",
+      "file_one": "file",
+      "file_other": "files",
+      "table_of_contents": "table of contents",
+      "selection": "selection"
```

Comment:
Even if you're just using one, I think it makes sense to add both the singular and plural to glossary with common entities. I imagine "selections" is used somewhere in Manifold.